packages/eslint-config-next/index.js

```diff
@@ -107,7 +107,13 @@ module.exports = {
       version: 'detect',
     },
     'import/parsers': {
-      [require.resolve('@typescript-eslint/parser')]: ['.ts', '.tsx', '.d.ts'],
+      [require.resolve('@typescript-eslint/parser')]: [
+        '.ts',
+        '.mts',
```

Comment:
Could you please provide the use case when you need to do linting of mts, mjs files in next js app src?

Reply:
Not sure if I understand the question correctly. As far as I know this line is needed to enable linting of files that has extension of ES module when running `next lint` command by default.